app/models/dialog_field_importer.rb

```diff
@@ -31,13 +31,17 @@ def import_field(dialog_field_attributes)
   private
 
   def set_category_for_tag_control(dialog_field, dialog_field_attributes)
-    category_name = dialog_field_attributes["options"][:category_name]
-    if category_name
-      category_description = dialog_field_attributes["options"][:category_description]
-      category = Category.find_by_name(category_name)
-      dialog_field.category = category.try(:description) == category_description ? category.id.to_s : nil
-    else
-      dialog_field.category = dialog_field_attributes["options"][:category_id]
-    end
+    dialog_field.category = adjust_category(dialog_field_attributes['options'])
+  end
+
+  def adjust_category(opts)
+    return nil if opts[:category_description].nil?
+    description = opts[:category_description]
+    real_category = if opts[:category_id] && description
```

Comment:
Thinking about the import operation and where these values come from the two variables `description` and `real_category` are not all that intuitive.

What do you think about: `import_description` and `import_category`?

Reply:
I like `import_category`. 

My thought on 'description` was to use that name instead of `category_description`, to differentiate that we need a `description` as our control element. Without a description there is no category to adjust for. Though, if that logic were to follow through, then `category` should suffice as opposed to `import_category` and the circle of discussion continues.

Reply:
Discussed further with @syncrou and agree that `category` (instead of `real_category`) is a better variable name here.